src/Json.re

```diff
@@ -347,10 +357,7 @@ and parseArrayValue = (text, pos) => {
     }
   }
   | ']' => ([value], pos + 1)
-  | _ => {
-    let (rest, pos) = parseArrayValue(text, pos);
-    ([value, ...rest], pos)
-  }
+  | _ => fail(text, pos, "unexpected character")
```

Comment:
oh, now I understand -- you're saying that the parser so far was too lax :) actually I'm totally fine with that (given that I'm parsing comments, which aren't technically valid json).

Reply:
Yeah, I'm all for the more lax parsing, but I think things like `[1 2 3 4]` maybe is taking it too far, or?

Reply:
yeah, that's fine :) as long as this doesn't break any of the current tests

Reply:
All your existing tests will be good in this. But I've changed some in the BigList to be more lax.